habitat_baselines/common/policy_registry.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+r"""PolicyRegistry is extended from habitat.Registry to provide
+registration for policy, while keeping Registry in habitat core intact.
+
+Import the policy registry object using
+
+.. code:: py
+
+    from habitat_baselines.common.policy_registry import policy_registry
+
+Decorator for registry different kind of classes with unique keys
+
+-   Register a policy: ``@policy_registry.register_policy``
+"""
+
+from typing import Optional
+
+from habitat.core.registry import Registry
+
+
+class PolicyRegistry(Registry):
```

Comment:
This doesn't need to be a separate class.  The two methods can be added to the existing BaselineRegistry.  The `"policy"` arg used bellow acts as an additional key so it won't conflict with anything else there.

Reply:
Done!